services/core-web/src/components/Forms/projectSummaries/ProjectSummaryForm.js

```diff
@@ -0,0 +1,263 @@
+                label="Project number"
+                component={renderConfig.FIELD}
+                disabled
+              />
+            </Form.Item>
+            <Form.Item>
+              <Field
+                id="project_summary_description"
+                name="project_summary_description"
+                label="Executive summary"
+                component={renderConfig.AUTO_SIZE_FIELD}
+                disabled
+              />
+            </Form.Item>
+          </Col>
+        </Row>
+      </div>
+    );
+  };
+
+  const renderAuthorizationsInvolved = () => {
+    const {
+      initialValues: { authorizations },
+      projectSummaryAuthorizationTypes,
+      projectSummaryPermitTypesHash,
+    } = props;
+    const transformedAuthorizationTypesHash = transformAuthorizationTypes(
+      projectSummaryAuthorizationTypes
+    );
+    // We need to make sure we only add parent authorization type labels once to the form
+    const parentHeadersAdded = [];
+    return (
+      <div id="authorizations-involved">
+        <Typography.Title level={3}>Authorizations involved</Typography.Title>
+        <p>
+          These are the authorizations the proponent believes may be needed for this project.
+          Additional authorizations may be required.
+        </p>
+        <br />
+        {authorizations.length > 0 &&
+          authorizations.map((a) => {
+            const parentCode =
+              transformedAuthorizationTypesHash[a.project_summary_authorization_type]?.parent?.code;
+            const parentHeaderAdded = parentHeadersAdded.includes(parentCode);
+            return (
+              <>
+                {!parentHeaderAdded && parentHeadersAdded.push(parentCode) ? (
+                  <h2>
+                    {
+                      transformedAuthorizationTypesHash[a.project_summary_authorization_type]
+                        ?.parent?.description
+                    }
+                  </h2>
+                ) : null}
+                <h4 className="padding-sm--bottom">
+                  {
+                    transformedAuthorizationTypesHash[a.project_summary_authorization_type]
+                      ?.description
+                  }
+                </h4>
+                <p className="bold padding-sm--bottom">Types of permits</p>
+                {a.project_summary_permit_type.map((pt) => {
+                  return <p className="padding-md--left">{projectSummaryPermitTypesHash[pt]}</p>;
+                })}
+                <br />
+                <p className="bold padding-sm--bottom">Existing permit numbers involved</p>
+                {a.existing_permits_authorizations.map((epa) => {
+                  return <p className="padding-md--left">{epa}</p>;
+                })}
+                <br />
+              </>
+            );
+          })}
+      </div>
+    );
+  };
+
+  const renderContacts = () => {
+    const {
+      projectSummary: { contacts },
+    } = props;
+    return (
```

Comment:
destructuring will fail the linter - is the linter running? We can revisit that rule and disable it if required, I can't remember the reasons why we enforced it.

Reply:
Yeah I have been fighting with that, not sure why destructuring fails here as it is valid ES6 syntax. I think the rules may need to be updated, and perhaps we can add in null defaults on that to make it more secure.

Reply:
That works for me - do you want to add those linting rules in this PR or disable the rule for this situation?